src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let primaryCarCost = 0;
+
+  for (let i = 0; i < days; i++) {
+    primaryCarCost += 40;
```

Comment:
you do not need this loop

you can reach the same using days * 40
do not increase the complexity